media/pom.xml

```diff
@@ -31,7 +31,7 @@
 
   <groupId>io.wcm</groupId>
   <artifactId>io.wcm.handler.media</artifactId>
-  <version>1.8.3-SNAPSHOT</version>
+  <version>1.9.0-SNAPSHOT</version>
```

Comment:
it's not longer required to update the bundle version with the latest bnd baseline plugin configuration used in the current develop branch. please set back to 1.8.3-SNAPSHOT. using version 1.9.0 in the changelog is correct.